resources/views/layouts/app.blade.php

```diff
@@ -21,11 +21,12 @@
             @include('layouts.navigation')
 
             <!-- Page Heading -->
-            <header class="bg-white shadow">
+                {{-- original --}}
+            {{-- <header class="bg-white shadow">
                 <div class="max-w-7xl mx-auto py-6 px-4 sm:px-6 lg:px-8">
                     {{ $header }}
                 </div>
-            </header>
+            </header> --}}
```

Comment:
This was commented out to match the wireframe? ;D 

Just check if there are any other pages that are using this component and that might be passing the header slot.

Reply:
It was originally throwing an error inside movies.index which implemented app-layout. Have manage to fix it now (not sure exactly was throwing the error initially)